Ubuntu编译安卓库报错：build-android-arm64-v8a.sh-->

/sherpa-onnx/sherpa-onnx/csrc/online-transducer-model.cc:209:5: error: unknown type name 'Manager'
  209 |     Manager *mgr, const OnlineModelConfig &config);
 sherpa-onnx/sherpa-onnx/csrc/online-transducer-model.cc:186:17: error: no matching function for call to 'ReadFile'
  186 |   auto buffer = ReadFile(mgr, config.transducer.encoder);

--- a/sherpa-onnx/csrc/online-transducer-model.cc
+++ b/sherpa-onnx/csrc/online-transducer-model.cc
@@ -206,7 +206,7 @@ std::unique_ptr<OnlineTransducerModel> OnlineTransducerModel::Create(
 
 #if __ANDROID_API__ >= 9
 template std::unique_ptr<OnlineTransducerModel> OnlineTransducerModel::Create(
-    Manager *mgr, const OnlineModelConfig &config);
+    AAssetManager *mgr, const OnlineModelConfig &config);
 #endif
 
 #if __OHOS__

